libc/src/__support/HashTable/sse2/bitmask_impl.inc

```diff
@@ -0,0 +1,40 @@
+//===-- HashTable BitMasks SSE2 Implementation ------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <emmintrin.h>
```

Comment:
Including `immintrin.h` instead should be more portable, even though you only need SSE2.